AttributeError: '_state' object has no attribute 'reset'

### Description

I tried to follow the guide at https://panel.holoviz.org/how_to/test/loadtests.html and got the error.

Maybe the problem is that I have an environment with weird versions?


### Reproduction steps

pip install panel pytest locust pytest-playwright pytest-asyncio loadwright==0.1.0
pytest test_loadwright.py

### Requirements

```bash
$ conda list
# packages in environment at /Users/cdeil/anaconda3/envs/hm-panel:
#
# Name                    Version                   Build  Channel
aiofiles                  22.1.0             pyhd8ed1ab_0    conda-forge
aiohttp                   3.8.5           py311heffc1b2_0    conda-forge
aiosignal                 1.3.1              pyhd8ed1ab_0    conda-forge
aiosqlite                 0.19.0             pyhd8ed1ab_0    conda-forge
altair                    5.1.1              pyhd8ed1ab_0    conda-forge
annotated-types           0.5.0              pyhd8ed1ab_0    conda-forge
anyio                     3.7.1              pyhd8ed1ab_0    conda-forge
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
argon2-cffi               23.1.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0          py311heffc1b2_4    conda-forge
arrow                     1.2.3              pyhd8ed1ab_0    conda-forge
asttokens                 2.4.0              pyhd8ed1ab_0    conda-forge
async-timeout             4.0.3              pyhd8ed1ab_0    conda-forge
attrs                     23.1.0             pyh71513ae_1    conda-forge
aws-c-auth                0.7.3                hab8b942_3    conda-forge
aws-c-cal                 0.6.2                he70778a_1    conda-forge
aws-c-common              0.9.0                hb547adb_0    conda-forge
aws-c-compression         0.2.17               he70778a_2    conda-forge
aws-c-event-stream        0.3.2                hcf14f3f_0    conda-forge
aws-c-http                0.7.12               he297698_1    conda-forge
aws-c-io                  0.13.32              h3c776e5_3    conda-forge
aws-c-mqtt                0.9.6                h83b98fe_0    conda-forge
aws-c-s3                  0.3.17               h24e141d_0    conda-forge
aws-c-sdkutils            0.1.12               he70778a_1    conda-forge
aws-checksums             0.1.17               he70778a_1    conda-forge
aws-crt-cpp               0.23.1               heded778_2    conda-forge
aws-sdk-cpp               1.11.156             h44dad36_2    conda-forge
babel                     2.12.1             pyhd8ed1ab_1    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                pyhd8ed1ab_3    conda-forge
backports.functools_lru_cache 1.6.5              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.12.2             pyha770c72_0    conda-forge
bleach                    6.0.0              pyhd8ed1ab_0    conda-forge
blinker                   1.6.3                    pypi_0    pypi
bokeh                     2.4.3                    pypi_0    pypi
boltons                   23.0.0          py311hca03da5_0  
brotli                    1.1.0                hb547adb_0    conda-forge
brotli-bin                1.1.0                hb547adb_0    conda-forge
brotli-python             1.1.0           py311ha891d26_0    conda-forge
bzip2                     1.0.8                h3422bc3_4    conda-forge
c-ares                    1.19.1               hb547adb_0    conda-forge
ca-certificates           2023.08.22           hca03da5_0  
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
cctools                   949.0.1             hc179dcd_25  
cctools_osx-arm64         949.0.1             h332cad3_25  
certifi                   2023.7.22       py311hca03da5_0  
cffi                      1.15.1          py311h4a08483_5    conda-forge
chardet                   4.0.0           py311hca03da5_1003  
charset-normalizer        3.2.0              pyhd8ed1ab_0    conda-forge
click                     8.1.7           unix_pyh707e725_0    conda-forge
cloudpickle               2.2.1              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
colorcet                  3.0.1              pyhd8ed1ab_0    conda-forge
comm                      0.1.4              pyhd8ed1ab_0    conda-forge
conda                     23.9.0          py311hca03da5_0  
conda-build               3.27.0          py311hca03da5_0  
conda-index               0.3.0           py311hca03da5_0  
conda-package-handling    2.2.0           py311hca03da5_0  
conda-package-streaming   0.9.0           py311hca03da5_0  
configargparse            1.7                      pypi_0    pypi
contourpy                 1.1.1           py311he4fd1f5_1    conda-forge
cryptography              41.0.3          py311hd4332d6_0  
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
dask-core                 2023.9.2           pyhd8ed1ab_0    conda-forge
datashader                0.15.2             pyhd8ed1ab_0    conda-forge
datashape                 0.5.4                      py_1    conda-forge
debugpy                   1.8.0           py311ha891d26_1    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
doit                      0.36.0          py311hca03da5_0  
duckdb                    0.8.2.dev5154            pypi_0    pypi
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
exceptiongroup            1.1.3              pyhd8ed1ab_0    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
fastapi                   0.103.1            pyhd8ed1ab_0    conda-forge
filelock                  3.9.0           py311hca03da5_0  
flask                     3.0.0                    pypi_0    pypi
flask-basicauth           0.2.0                    pypi_0    pypi
flask-cors                4.0.0                    pypi_0    pypi
fonttools                 4.42.1          py311heffc1b2_0    conda-forge
fqdn                      1.5.1              pyhd8ed1ab_0    conda-forge
freetype                  2.12.1               hadb7bae_2    conda-forge
frozenlist                1.4.0           py311heffc1b2_1    conda-forge
fsspec                    2023.9.2           pyh1a96a4e_0    conda-forge
gevent                    23.9.1                   pypi_0    pypi
geventhttpclient          2.0.11                   pypi_0    pypi
gflags                    2.2.2             hc88da5d_1004    conda-forge
glog                      0.6.0                h6da1cb0_0    conda-forge
greenlet                  3.0.0                    pypi_0    pypi
h11                       0.14.0             pyhd8ed1ab_0    conda-forge
h2                        4.1.0              pyhd8ed1ab_0    conda-forge
holoviews                 1.18.0a2                   py_0    pyviz/label/dev
hpack                     4.0.0              pyh9f0ad1d_0    conda-forge
httpcore                  0.18.0             pyhd8ed1ab_0    conda-forge
httpx                     0.25.0             pyhd8ed1ab_0    conda-forge
hvplot                    0.9.0a2                    py_0    pyviz/label/dev
hyperframe                6.0.1              pyhd8ed1ab_0    conda-forge
icu                       68.1                 hc377ac9_0  
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.8.0              pyha770c72_0    conda-forge
importlib_metadata        6.8.0                hd8ed1ab_0    conda-forge
importlib_resources       6.1.0              pyhd8ed1ab_0    conda-forge
iniconfig                 2.0.0              pyhd8ed1ab_0    conda-forge
ipykernel                 6.25.2             pyh1050b4e_0    conda-forge
ipython                   8.15.0             pyh31c8845_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                8.1.1              pyhd8ed1ab_0    conda-forge
isoduration               20.11.0            pyhd8ed1ab_0    conda-forge
itsdangerous              2.1.2                    pypi_0    pypi
jedi                      0.19.0             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
joblib                    1.3.2              pyhd8ed1ab_0    conda-forge
json5                     0.9.14             pyhd8ed1ab_0    conda-forge
jsonpatch                 1.32               pyhd3eb1b0_0  
jsonpointer               2.4             py311h267d04e_3    conda-forge
jsonschema                4.19.1             pyhd8ed1ab_0    conda-forge
jsonschema-specifications 2023.7.1           pyhd8ed1ab_0    conda-forge
jsonschema-with-format-nongpl 4.19.1             pyhd8ed1ab_0    conda-forge
jupyter_bokeh             3.0.7              pyhd8ed1ab_0    conda-forge
jupyter_client            7.4.9              pyhd8ed1ab_0    conda-forge
jupyter_core              5.3.1           py311h267d04e_1    conda-forge
jupyter_events            0.7.0              pyhd8ed1ab_2    conda-forge
jupyter_server            2.7.3              pyhd8ed1ab_1    conda-forge
jupyter_server_fileid     0.9.0              pyhd8ed1ab_0    conda-forge
jupyter_server_terminals  0.4.4              pyhd8ed1ab_1    conda-forge
jupyter_server_ydoc       0.8.0              pyhd8ed1ab_0    conda-forge
jupyter_ydoc              0.2.4              pyhd8ed1ab_0    conda-forge
jupyterlab                3.6.6              pyhd8ed1ab_0    conda-forge
jupyterlab-myst           1.2.0              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jupyterlab_server         2.25.0             pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        3.0.9              pyhd8ed1ab_0    conda-forge
kiwisolver                1.4.5           py311he4fd1f5_1    conda-forge
krb5                      1.21.2               h92f50d5_0    conda-forge
lcms2                     2.15                 h40e5a24_2    conda-forge
ld64                      530                 hb29bf3f_25  
ld64_osx-arm64            530                 h001ce53_25  
ldid                      2.1.5                h20b2a84_3  
lerc                      4.0.0                h9a09cb3_0    conda-forge
libabseil                 20230802.1      cxx17_h13dd4ca_0    conda-forge
libarchive                3.6.1                he3a3bf9_0  
libarrow                  13.0.0           h0c25ffe_4_cpu    conda-forge
libblas                   3.9.0           18_osxarm64_openblas    conda-forge
libbrotlicommon           1.1.0                hb547adb_0    conda-forge
libbrotlidec              1.1.0                hb547adb_0    conda-forge
libbrotlienc              1.1.0                hb547adb_0    conda-forge
libcblas                  3.9.0           18_osxarm64_openblas    conda-forge
libcrc32c                 1.1.2                hbdafb3b_0    conda-forge
libcurl                   8.3.0                hc52a3a8_0    conda-forge
libcxx                    16.0.6               h4653b0c_0    conda-forge
libdeflate                1.19                 hb547adb_0    conda-forge
libedit                   3.1.20191231         hc8eb9b7_2    conda-forge
libev                     4.33                 h642e427_1    conda-forge
libevent                  2.1.12               h2757513_1    conda-forge
libexpat                  2.5.0                hb7217d7_1    conda-forge
libffi                    3.4.2                h3422bc3_5    conda-forge
libgfortran               5.0.0           13_2_0_hd922786_1    conda-forge
libgfortran5              13.2.0               hf226fd6_1    conda-forge
libgoogle-cloud           2.12.0               hde42cda_2    conda-forge
libgrpc                   1.57.0               hdbe17d8_1    conda-forge
libiconv                  1.16                 h1a28f6b_2  
libjpeg-turbo             2.1.5.1              hb547adb_1    conda-forge
liblapack                 3.9.0           18_osxarm64_openblas    conda-forge
liblief                   0.12.3               h313beb8_0  
libllvm14                 14.0.6               hd1a9a77_4    conda-forge
libnghttp2                1.52.0               hae82a92_0    conda-forge
libopenblas               0.3.24          openmp_hd76b1f2_0    conda-forge
libpng                    1.6.39               h76d750c_0    conda-forge
libprotobuf               4.23.4               hf590ac1_6    conda-forge
libsodium                 1.0.18               h27ca646_1    conda-forge
libsqlite                 3.43.0               hb31c410_0    conda-forge
libssh2                   1.11.0               h7a5bd25_0    conda-forge
libthrift                 0.19.0               ha061701_0    conda-forge
libtiff                   4.6.0                h77c4dce_1    conda-forge
libutf8proc               2.8.0                h1a8c8d9_0    conda-forge
libwebp-base              1.3.2                hb547adb_0    conda-forge
libxcb                    1.15                 hf346824_0    conda-forge
libxml2                   2.9.14               h8c5e841_0  
libzlib                   1.2.13               h53f4e23_5    conda-forge
linkify-it-py             2.0.0              pyhd8ed1ab_0    conda-forge
llvm-openmp               16.0.6               h1c12783_0    conda-forge
llvmlite                  0.40.1          py311hea943cd_0    conda-forge
loadwright                0.1.0                    pypi_0    pypi
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
locust                    2.17.0                   pypi_0    pypi
lz4-c                     1.9.4                hb7217d7_0    conda-forge
markdown                  3.4.4              pyhd8ed1ab_0    conda-forge
markdown-it-py            3.0.0              pyhd8ed1ab_0    conda-forge
markupsafe                2.1.3           py311heffc1b2_1    conda-forge
matplotlib                3.8.0           py311ha1ab1f8_1    conda-forge
matplotlib-base           3.8.0           py311h3bc9839_1    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mdit-py-plugins           0.4.0              pyhd8ed1ab_0    conda-forge
mdurl                     0.1.0              pyhd8ed1ab_0    conda-forge
mistune                   3.0.1              pyhd8ed1ab_0    conda-forge
more-itertools            8.12.0             pyhd3eb1b0_0  
msgpack                   1.0.7                    pypi_0    pypi
multidict                 6.0.4           py311he2be06e_0    conda-forge
multipledispatch          0.6.0                      py_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
nbclassic                 1.0.0              pyhb4ecaf3_1    conda-forge
nbclient                  0.8.0              pyhd8ed1ab_0    conda-forge
nbconvert                 7.8.0              pyhd8ed1ab_0    conda-forge
nbconvert-core            7.8.0              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.8.0              pyhd8ed1ab_0    conda-forge
nbformat                  5.9.2              pyhd8ed1ab_0    conda-forge
ncurses                   6.4                  h7ea286d_0    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
notebook                  6.5.6              pyha770c72_0    conda-forge
notebook-shim             0.2.3              pyhd8ed1ab_0    conda-forge
numba                     0.57.1          py311hbf3c4e2_0    conda-forge
numpy                     1.24.4          py311hb8f3215_0    conda-forge
openjpeg                  2.5.0                h4c1507b_3    conda-forge
openssl                   3.1.3                h53f4e23_0    conda-forge
orc                       1.9.0                hbfdecac_2    conda-forge
overrides                 7.4.0              pyhd8ed1ab_0    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    2.0.3           py311h9e438b8_1    conda-forge
pandoc                    3.1.3                hce30654_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
panel                     0.14.4                   pypi_0    pypi
param                     2.0.0rc7                 pypi_0    pypi
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
partd                     1.4.1              pyhd8ed1ab_0    conda-forge
patch                     2.7.6             h1a28f6b_1001  
patsy                     0.5.3              pyhd8ed1ab_0    conda-forge
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    10.0.1          py311he9c13d2_1    conda-forge
pip                       23.2.1             pyhd8ed1ab_0    conda-forge
pkginfo                   1.9.6           py311hca03da5_0  
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_1    conda-forge
platformdirs              3.10.0             pyhd8ed1ab_0    conda-forge
playwright                1.39.0                   pypi_0    pypi
pluggy                    1.3.0              pyhd8ed1ab_0    conda-forge
prometheus_client         0.17.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.39             pyha770c72_0    conda-forge
prompt_toolkit            3.0.39               hd8ed1ab_0    conda-forge
psutil                    5.9.5           py311heffc1b2_1    conda-forge
pthread-stubs             0.4               h27ca646_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py                        1.11.0             pyhd3eb1b0_0  
py-lief                   0.12.3          py311h313beb8_0  
pyarrow                   13.0.0          py311hd7bc329_4_cpu    conda-forge
pycosat                   0.6.6           py311h80987f9_0  
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pyct                      0.4.6                      py_0    conda-forge
pyct-core                 0.4.6                      py_0    conda-forge
pyctdev                   0.7.0                      py_0    pyviz
pydantic                  2.4.1              pyhd8ed1ab_0    conda-forge
pydantic-core             2.10.1          py311h0563b04_0    conda-forge
pyee                      11.0.1                   pypi_0    pypi
pygments                  2.16.1             pyhd8ed1ab_0    conda-forge
pyinstrument              4.5.3           py311heffc1b2_1    conda-forge
pyjwt                     2.8.0                    pypi_0    pypi
pyobjc-core               10.0            py311hb702dc4_0    conda-forge
pyobjc-framework-cocoa    10.0            py311hb702dc4_1    conda-forge
pyopenssl                 23.2.0          py311hca03da5_0  
pyparsing                 3.1.1              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
pytest                    7.4.2              pyhd8ed1ab_0    conda-forge
pytest-asyncio            0.21.1                   pypi_0    pypi
pytest-base-url           2.0.0                    pypi_0    pypi
pytest-playwright         0.4.3                    pypi_0    pypi
python                    3.11.5          h47c9636_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.18.0             pyhd8ed1ab_0    conda-forge
python-json-logger        2.0.7              pyhd8ed1ab_0    conda-forge
python-libarchive-c       2.9                pyhd3eb1b0_1  
python-slugify            8.0.1                    pypi_0    pypi
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python_abi                3.11                    4_cp311    conda-forge
pytz                      2023.3.post1       pyhd8ed1ab_0    conda-forge
pyviz_comms               2.3.2              pyhd8ed1ab_0    conda-forge
pyyaml                    6.0.1           py311heffc1b2_1    conda-forge
pyzmq                     24.0.1          py311h0f351f6_1    conda-forge
re2                       2023.03.02           hc5e2d97_0    conda-forge
readline                  8.2                  h92ec313_1    conda-forge
referencing               0.30.2             pyhd8ed1ab_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
rfc3339-validator         0.1.4              pyhd8ed1ab_0    conda-forge
rfc3986-validator         0.1.1              pyh9f0ad1d_0    conda-forge
roundrobin                0.0.4                    pypi_0    pypi
rpds-py                   0.10.3          py311h0563b04_0    conda-forge
ruamel.yaml               0.17.21         py311h80987f9_0  
scikit-learn              1.3.1           py311hf0b18b8_0    conda-forge
scipy                     1.11.2          py311h93d07a4_1    conda-forge
seaborn                   0.12.2               hd8ed1ab_0    conda-forge
seaborn-base              0.12.2             pyhd8ed1ab_0    conda-forge
send2trash                1.8.2              pyhd1c38e8_0    conda-forge
setuptools                68.2.2             pyhd8ed1ab_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snakeviz                  2.2.0              pyhd8ed1ab_0    conda-forge
snappy                    1.1.10               h17c5cce_0    conda-forge
sniffio                   1.3.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.5                pyhd8ed1ab_1    conda-forge
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
starlette                 0.27.0             pyhd8ed1ab_0    conda-forge
statsmodels               0.14.0          py311hb49d859_1    conda-forge
tapi                      1100.0.11            h8754e6a_1  
terminado                 0.17.1             pyhd1c38e8_0    conda-forge
text-unidecode            1.3                      pypi_0    pypi
threadpoolctl             3.2.0              pyha21a80b_0    conda-forge
tinycss2                  1.2.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.13               hb31c410_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tornado                   6.3.3           py311heffc1b2_1    conda-forge
tqdm                      4.66.1             pyhd8ed1ab_0    conda-forge
traitlets                 5.10.1             pyhd8ed1ab_0    conda-forge
truststore                0.8.0           py311hca03da5_0  
typing-extensions         4.8.0                hd8ed1ab_0    conda-forge
typing_extensions         4.8.0              pyha770c72_0    conda-forge
typing_utils              0.1.0              pyhd8ed1ab_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
uc-micro-py               1.0.1              pyhd8ed1ab_0    conda-forge
uri-template              1.3.0              pyhd8ed1ab_0    conda-forge
urllib3                   2.0.5              pyhd8ed1ab_0    conda-forge
uvicorn                   0.23.2          py311h267d04e_1    conda-forge
wcwidth                   0.2.6              pyhd8ed1ab_0    conda-forge
webcolors                 1.13               pyhd8ed1ab_0    conda-forge
webencodings              0.5.1              pyhd8ed1ab_2    conda-forge
websocket-client          1.6.3              pyhd8ed1ab_0    conda-forge
werkzeug                  3.0.0                    pypi_0    pypi
wheel                     0.41.2             pyhd8ed1ab_0    conda-forge
widgetsnbextension        4.0.9              pyhd8ed1ab_0    conda-forge
xarray                    2023.9.0           pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.11               hb547adb_0    conda-forge
xorg-libxdmcp             1.1.3                h27ca646_0    conda-forge
xyzservices               2023.7.0           pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h57fd34a_0    conda-forge
y-py                      0.5.9           py311heb423af_0    conda-forge
yaml                      0.2.5                h3422bc3_2    conda-forge
yarl                      1.9.2           py311heffc1b2_0    conda-forge
ypy-websocket             0.8.2              pyhd8ed1ab_0    conda-forge
zeromq                    4.3.4                hbdafb3b_1    conda-forge
zipp                      3.17.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               h53f4e23_5    conda-forge
zope-event                5.0                      pypi_0    pypi
zope-interface            6.1                      pypi_0    pypi
zstandard                 0.19.0          py311h80987f9_0  
zstd                      1.5.5                h4f39d0f_0    conda-forge
```


### Code

_No response_

### Logs

```bash
$ pytest test_loadwright.py 
=========================================== test session starts ===========================================
platform darwin -- Python 3.11.5, pytest-7.4.2, pluggy-1.3.0
rootdir: /Users/cdeil/temp/playwright
plugins: playwright-0.4.3, asyncio-0.21.1, base-url-2.0.0, anyio-3.7.1
asyncio: mode=Mode.STRICT
collected 1 item                                                                                          

test_loadwright.py .E                                                                               [100%]

================================================= ERRORS ==================================================
__________________________________ ERROR at teardown of test_component_2 __________________________________

    @pytest.fixture(autouse=True)
    def server_cleanup():
        """
        Clean up server state after each test.
        """
        try:
            yield
        finally:
>           pn.state.reset()
E           AttributeError: '_state' object has no attribute 'reset'

conftest.py:22: AttributeError
------------------------------------------ Captured stdout call -------------------------------------------
Launching server at http://localhost:6001
------------------------------------------ Captured stderr call -------------------------------------------
WARNING:bokeh.core.validation.check:W-1005 (FIXED_SIZING_MODE): 'fixed' sizing mode requires width and height to be set: Button(id='1004', ...)
WARNING:bokeh.core.validation.check:W-1005 (FIXED_SIZING_MODE): 'fixed' sizing mode requires width and height to be set: Button(id='1012', ...)
WARNING:bokeh.core.validation.check:W-1005 (FIXED_SIZING_MODE): 'fixed' sizing mode requires width and height to be set: Button(id='1020', ...)
WARNING:bokeh.core.validation.check:W-1005 (FIXED_SIZING_MODE): 'fixed' sizing mode requires width and height to be set: Button(id='1064', ...)
WARNING:bokeh.core.validation.check:W-1005 (FIXED_SIZING_MODE): 'fixed' sizing mode requires width and height to be set: Button(id='1087', ...)
-------------------------------------------- Captured log call --------------------------------------------
WARNING  tornado.access:web.py:2344 404 GET /favicon.ico (::1) 0.23ms
============================================ warnings summary =============================================
../../anaconda3/envs/hm-panel/lib/python3.11/site-packages/bokeh/core/property/primitive.py:37
  /Users/cdeil/anaconda3/envs/hm-panel/lib/python3.11/site-packages/bokeh/core/property/primitive.py:37: DeprecationWarning: `np.bool8` is a deprecated alias for `np.bool_`.  (Deprecated NumPy 1.24)
    bokeh_bool_types += (np.bool8,)

test_loadwright.py::test_component_2
  /Users/cdeil/anaconda3/envs/hm-panel/lib/python3.11/site-packages/panel/io/rest.py:11: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    import pkg_resources

test_loadwright.py::test_component_2
  /Users/cdeil/anaconda3/envs/hm-panel/lib/python3.11/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('ruamel')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_loadwright.py::test_component_2
test_loadwright.py::test_component_2
  /Users/cdeil/anaconda3/envs/hm-panel/lib/python3.11/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('zope')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_loadwright.py::test_component_2
  /Users/cdeil/anaconda3/envs/hm-panel/lib/python3.11/site-packages/panel/io/server.py:911: DeprecationWarning: make_current is deprecated; start the event loop first
    loop.make_current()

test_loadwright.py::test_component_2
  /Users/cdeil/anaconda3/envs/hm-panel/lib/python3.11/site-packages/pytest_asyncio/plugin.py:451: DeprecationWarning: pytest-asyncio detected an unclosed event loop when tearing down the event_loop
  fixture: <_UnixSelectorEventLoop running=False closed=False debug=False>
  pytest-asyncio will close the event loop for you, but future versions of the
  library will no longer do so. In order to ensure compatibility with future
  versions, please make sure that:
      1. Any custom "event_loop" fixture properly closes the loop after yielding it
      2. The scopes of your custom "event_loop" fixtures do not overlap
      3. Your code does not modify the event loop in async fixtures or tests
  
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
========================================= short test summary info =========================================
ERROR test_loadwright.py::test_component_2 - AttributeError: '_state' object has no attribute 'reset'
================================= 1 passed, 7 warnings, 1 error in 24.75s =================================
(
```


### Additional Context

_No response_

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,8 +34,8 @@ classifiers = [
     "Topic :: Software Development :: Libraries",
 ]
 dependencies = [
-    "bokeh<3.0",
-    "panel<1.0",
+    "bokeh",
+    "panel",
     "hvplot",
     "holoviews",
     "pandas",

--- a/src/loadwright/__init__.py
+++ b/src/loadwright/__init__.py
@@ -4,6 +4,6 @@ from loadwright.runner import LoadTestRunner
 from loadwright.user import User
 from loadwright.viewer import LoadTestViewer
 
-VERSION = "0.1.0"
+VERSION = "0.2.0"
 
 __all__ = ["read_loadwright_file", "LoadTestRunner", "User", "LoadTestViewer"]

